documentation/examples/prometheus.yml

```diff
@@ -26,4 +26,5 @@ scrape_configs:
     # scheme defaults to 'http'.
 
     static_configs:
-    - targets: ['localhost:9090']
+    - targets:
```

Comment:
Why are you changing this? Generally, teaching users to put things with colons in quotes is a good idea.